server/lib/vertex_tryon_client.py

```
@@ -0,0 +1,120 @@
+                    }
+                }
+                for b in item_image_bytes
+            ],
+        }
+    )
+    return instance
+
+
+def _decode_first_prediction(response) -> bytes:
+    if not response.predictions:
+        raise VertexTryOnError("vertex returned no predictions")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)